Mono.Cecil/SecurityDeclaration.cs

```diff
@@ -70,6 +70,14 @@ public SecurityAttribute (TypeReference attributeType)
 		{
 			this.attribute_type = attributeType;
 		}
+
+		bool ICustomAttribute.HasConstructorArguments {
+			get { return false; }
+		}
+
+		Collection<CustomAttributeArgument> ICustomAttribute.ConstructorArguments {
+			get { return new Collection<CustomAttributeArgument> (); }
```

Comment:
Could you throw a NotSupportedException here?

Reply:
Changed: e2176ecceba35a34c3bd5910942e22bd8f5fed14